src/test/java/integration/android/ContextTest.java

```diff
@@ -0,0 +1,24 @@
+package integration.android;
+
+import com.codeborne.selenide.appium.SelenideAppium;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static com.codeborne.selenide.WebDriverRunner.closeWebDriver;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ContextTest extends BaseSwagLabsAndroidTest {
+
+  @BeforeEach
+  void setUp() {
+    closeWebDriver();
+  }
+
+  @Test
+  void contexts() {
+    SelenideAppium.openAndroidDeepLink("mydemoapprn://webview", "com.saucelabs.mydemoapp.rn");
+    SelenideAppium.setContext("NATIVE_APP");
```

Comment:
It would be great to test switching between _different contexts_.
Does this app has some other contexts in addition to `NATIVE_APP`?

Reply:
@asolntsev - It does not have other contexts to switch to and test